### DISCUSSION TITLE
Getting GPU render on the windows side ..

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4434] [Ideas]
Scott has mentioned about changing up the ffmpeg to make use of GPU .. if anyone has tips/hints/links to make this happen please feel free to add them here.

**Comment by blun23** [comment]
I would love to see what my tower could do up against my mac mini m2.  This would also help stop all the bickering in the community and probably save alot of people from having to buy new pc's to keep up.

**Comment by cbredlow** [comment]
Is it possible to implement something with OpenCL to utilize CPU/GPU?

**Comment by derwin12** [comment]
OpenCL looks interesting .. Scott mentioned Vulkan but that appears to be just speeding up graphics stuff nothing on the compute side where xlights renders suffer the most. Ideally want it to be opensource so it doesnt just go away..

**Comment by cbredlow** [comment]
Yeah, I looked into that too. I'd offer to help, but not super familiar with C++ to try and do it myself. Though willing to learn it.

**Comment by MovieDad** [comment]
This looks to be implemented now using FFMPEG and DirectX, been playing with 2024.8.1 and trying the different options, only seeing very small changes in performance, around 1 second on average between all options but CUDA as I am using AMD. Even turning off the feature is showing about the same, is there something else I should be doing to help test this? Another thing I noticed is that when rending videos I am not seeing any GPU load at all. I am happy to post videos of tests etc as well just not sure where to do that.

**Comment by derwin12** [comment]
I have seen similar results .. I believe this only impacts the video effect and only on reads.
